browser-extension/src/utils.js

```diff
@@ -89,6 +89,25 @@ export const addRateLater = async (video_id) => {
   };
 };
 
+export const addRateLaterBulk = async (videoIds) => {
+  const ratingStatusReponse = await fetchTournesolApi(
+    'users/me/rate_later/videos/_bulk_create',
```

Comment:
Add trailing slash to avoid unnecessary redirection:
```suggestion
    'users/me/rate_later/videos/_bulk_create/',
```